packages/bsky/src/migrate-moderation-data.ts

```diff
@@ -272,9 +285,29 @@ export async function MigrateModerationData() {
 
   const primaryDb = db.getPrimary()
 
-  const counts = await countEntries(primaryDb)
-  const totalEntries = counts.actionsCount + counts.reportsCount
+  const [counts, existingEventsCount] = await Promise.all([
+    countEntries(primaryDb),
+    countEvents(primaryDb),
+  ])
+
+  // If there are existing events in the moderation_event table, we assume that the migration has already been run
+  // so we just bring over any new reports since last run
+  if (existingEventsCount) {
+    console.log(
+      `Found ${existingEventsCount} existing events. Migrating ${counts.reportsCount} reports only, ignoring actions`,
+    )
+    const reportMigrationStartedAt = Date.now()
+    // TODO: Make sure to set the appropriate last report id to ensure continuity before running the script
+    await createEvents(primaryDb, { onlyReportsAboveId: 1000000 })
```

Comment:
@devinivy this is hardcoded so you'd have to set the id before running the script. I could make it a cli arg but not really sure how you're planning on running this.